src/neural_data_simulator/core/ephys_generator.py

```diff
@@ -800,11 +685,12 @@ def stop(self):
 
     def _execute(self):
         self._update_spike_rates()
+        time_now = self._timer.total_elapsed_time()
+
         if self._last_output_time is None:
-            self._last_output_time = pylsl.local_clock()
```

Comment:
@florin-pop or @d-lucena do you know why `pylsl.local_clock` was used here in `ephys_generator.py` originally instead of the `Timer` class?

* Or, if it is important to use `pylsl.local_clock`, should we change the `Timer` class to use `pylsl.local_clock` (instead of perf counter)

Reply:
`local_clock` can help for aligning/syncing across computers.